apps/builder/app/canvas/shared/use-pointer-outline.ts

```diff
@@ -18,6 +18,7 @@ export const usePointerOutline = () => {
     ref.current = div;
```

Comment:
Is this module even used?

Reply:
I used it in the past to debug the coordinates of the pointer, since we are doing recalculation based on the canvas scale and position, we basically need to adjust the pointer position that actually is on canvas during drag&drop

I used to turn this on to draw a rectangle for the pointer, we haven't had issues for a while, but it's an easy way to check where the pointer coordinate would be during drag to see if we are sending wrong coordinates to the canvas

Reply:
it could use a better place on the file system, like some kind of tools folder or something, if we want to keep it